idb/idb.go

```diff
@@ -401,3 +408,11 @@ type Health struct {
 type NetworkState struct {
 	GenesisHash crypto.Digest `codec:"genesis-hash"`
 }
+
+// MaxTransactionsError records the error when transaction counts exceeds MaxTransactionsLimit.
+type MaxTransactionsError struct {
+}
+
+func (e MaxTransactionsError) Error() string {
+	return "Max transactions limit exceeded. header-only flag should be enabled"
```

Comment:
Could you put this string in `error_messages.go`?

Reply:
added.